app/models/concerns/fixerable.rb

```diff
@@ -137,13 +137,14 @@ def fixerable_final_path
 
   # temporary location
   def fixerable_temp_url(options = {})
-    uri = URI.parse(fixtemp)
+    value = fixtemp.include?('://') ? fixtemp : "s3://#{ENV['AWS_BUCKET']}/#{value}"
```

Comment:
Exchange (v3) uploads audio_files with a upload_path set to `path/to/file.mp3`.  But CMS (v4) uses the full `s3://bucket/path/to/file.mp3`.  Was getting some `no method 'starts_with?' for nil` error below, since the parsed value has no scheme.

This adds the mediajoint production s3 bucket prefix to those relative paths.

Reply:
Looks good, maybe worth a comment?

Reply:
Yeah, I'll make this more explicit.